### PR TITLE
Top movies endpoint

### PR DESCRIPTION
**Pull request by reshav-abraham**
Setup the react client and axios config. Changed Cors setting django

### CODE REVIEW

**Comment by reshav-abraham** [comment]
Stacked on the first pr

File: client/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,5912 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (fkotsian) [inline client/yarn.lock:1]: love yarn

File: movie_api/core/views.py [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
 
 import os
 import logging
 
+log = logging.getLogger(__name__)
+
 from movie_client import MovieClient
 
 if "API_KEY" in os.environ:
     movie_client = MovieClient(os.environ['API_KEY'])
```
Comment (fkotsian) [inline movie_api/core/views.py:13]: you could move this conditional into the movie_client constructor if you want but I'm not opposed to it at this stage 👍

File: movie_api/movie_api/urls.py [hunk 3]
```diff
@@ -5,4 +5,5 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', views.index, name='index'),
+    path('getTopMovies/', views.get_top_movies),
```
Comment (fkotsian) [inline movie_api/movie_api/urls.py:8]: route

File: client/index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import { getTopMovies } from './api/MovieApi';
+// check for existing cards
+
+function App() {
+
+  return (
+    <div>HELLO</div>
+  );
+}
+{getTopMovies()}
```
Comment (fkotsian) [inline client/index.js:14]: render movies 🔥

File: client/api/MovieApi.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+const api = axios.create({
+    baseURL:"http://localhost:8000"
+});
+
+export function getTopMovies(setPlanData) {
+    let res = api.get('/getTopMovies/');
```
Comment (fkotsian) [inline client/api/MovieApi.js:8]: apiClient

File: movie_api/movie_client/movie_client.py [hunk 6]
```diff
@@ -16,4 +22,12 @@ def __init__(self, api_key):
 
     def validate_key(self):
         r = requests.get(f'{self.url}550?api_key={self.api_key}')
-        return r.status_code 
\ No newline at end of file
+        return r.status_code
+
+    def get_top_movies(self):
+        r = requests.get(f"{self.url}top_rated?api_key={self.api_key}")
```
Comment (fkotsian) [inline movie_api/movie_client/movie_client.py:28]: passes API_KEY in query params 👍

File: client/index.html [hunk 7]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>To Do list and Planner</title>
+    <title>Movie App</title>
```
Comment (fkotsian) [inline client/index.html:5]: Titles!! 🔥 🔥

**Review by fkotsian — APPROVED** [review]
Looking good dude! I like the client/API split - leaves open muy testability in the future 👍

**Pull request merged**